Snowcast/ResortsTableViewController.swift

```diff
@@ -30,17 +48,25 @@ class ResortsTableViewController: UITableViewController {
     }
 
     override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
-        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath)
+        let cell = tableView.dequeueReusableCellWithIdentifier(resortTableCell, forIndexPath: indexPath) as! ResortTableViewCell
```

Comment:
Length Violation: Line should be 100 characters or less: currently 129 characters